src/tox/venv.py

```diff
@@ -515,6 +515,11 @@ def _pcall(
         env = self._get_os_environ(is_test_command=is_test_command)
         bin_dir = str(self.envconfig.envbindir)
         env["PATH"] = os.pathsep.join([bin_dir, os.environ["PATH"]])
+        if "PATH" in self.envconfig.setenv:
+            env["PATH"] = os.pathsep.join([env["PATH"], self.envconfig.setenv["PATH"]])
```

Comment:
this needs more thought, why join here? this so to say overrides the setenv 🤔 we should only join if the setenv includes the previous, not?

Reply:
Right, will have to do something about that.